src/main/java/emu/grasscutter/scripts/SceneScriptManager.java

```diff
@@ -407,6 +396,9 @@ public void callEvent(int eventType, ScriptArgs params) {
 			}
 			
 			if (ret.isboolean() && ret.checkboolean()) {
+				if(StringUtils.isEmpty(trigger.action)){
```

Comment:
Why are you using external library for such simple task? Java has native `.isEmpty()` for `String`

Reply:
here I want to check if it is null or empty, now apache commons is removed